flow/ProtocolVersion.h

```diff
@@ -73,6 +73,10 @@ class ProtocolVersion {
 	constexpr bool operator>(const ProtocolVersion other) const { return version() > other.version(); }
 
 public: // introduced features
+	// The 5th byte from right is dev versoin, for example, 2 in 0x0FDB00B061020000LL;
+	// It was used to identify a protocol change (e.g., interface change) between release versions (say 5.2.1 and 5.2.2)
+	// We stoped using the dev version at some point in the past.
+	// To ensure binaries work across patch releases (e.g., 6.2.1 to 6.2.22), we ensure no protocol change.
```

Comment:
```suggestion
	// To ensure binaries work across patch releases (e.g., 6.2.0 to 6.2.22), we require that the protocol version be the same for each of them.
```

Reply:
somehow, it cannot commit to batch. I modify it locally and put your name on the commit to avoid too many commits. :)